.circleci/config.yml

```diff
@@ -110,6 +110,10 @@ jobs:
                     libgtk-3-0 \
                     libgbm1
                   sudo add-apt-repository -y ppa:alexlarsson/flatpak
+      - run:
+          name: 'Install pnpm'
+          command: |
+            npm install -g pnpm@10.0.0
```

Comment:
Could also do this in `corepack` in theory, but I found this to be easier to grok.